common/static/common/js/discussion/utils.js

```diff
@@ -444,19 +463,52 @@
             }
         };
 
-        DiscussionUtil.abbreviateHTML = function(html, minLength) {
+        DiscussionUtil.convertMath = function(element, alreadyEscaped) {
+            var elementContent = element.html();
+            if (alreadyEscaped){
+                // markdownWithHighlight requires unescaped text, so we must unescape it.
+                // safe-lint: disable=javascript-escape
+                elementContent = _.unescape(edx.HtmlUtils.HTML(elementContent));
+            }
```

Comment:
@robrap No, I shouldn't have added `edx.HtmlUtils.HTML`, but it worked anyway and I haven't noticed it was there. And it worked exactly because call to `edx.HtmlUtils.HTML` was a no-op.

It wasn't needed, I just added it unintentionally (probably copied from some other place).